README.md

```diff
@@ -110,6 +110,12 @@ Below is an example of regexp patterns used for autolinking at https://community
 3. Using autolinking to create group mentions. Use (note that clicking the resulting at-mention redirects to a broken page):
   - Pattern: `@customgroup*`
   - Template: `[@customgroup]( \\* @user1 @user2 @user3 \\* )`
+  
+4. For servers with multiple domains (like community and community-daily on the offical Mattermost Server) a substition of absolute conversation links to relative links is recommended to prevent issues in the mobile app. Add one pattern for each domain Used:
```

Comment:
```suggestion
4. For servers with multiple domains (like community and community-daily on the [public Mattermost Server](https://community.mattermost)), a substitution of absolute conversation links to relative links is recommended to prevent issues in the mobile app. Add one pattern for each domain used:
```